src/matrix/storage/idb/stores/OperationStore.ts

```diff
@@ -62,12 +65,12 @@ export class OperationStore {
     }
 
     add(operation: Operation): void {
-        operation.scopeTypeKey = encodeScopeTypeKey(operation.scope, operation.type);
-        this._store.add(operation);
+        (operation as OperationEntry).scopeTypeKey = encodeScopeTypeKey(operation.scope, operation.type);
+        this._store.add(operation as OperationEntry);
     }
 
     update(operation: Operation): void {
-        this._store.put(operation);
+        this._store.put(operation as OperationEntry);
```

Comment:
Not sure about this here; nothing guarantees that the given `Operation` will have the `scopeTypeKey`.

Reply:
Indeed, but fine for now I think.